packages/datadog-instrumentations/src/jest.js

```diff
@@ -495,6 +495,8 @@ addHook({
       _ddTestModuleId,
       _ddTestSessionId,
       _ddTestCommand,
+      _ddForcedToRun,
+      _ddUnskippable,
```

Comment:
we're removing these values from the `testEnvironmentOptions` not to mess with jest caching mechanism, for the same reason we removed `_ddTestSessionId` and the others